README.md

```diff
@@ -9,6 +9,7 @@ A Kubernetes Operator based on the Operator SDK for creating and syncing resourc
 ## Help and Documentation
 
 The official documentation might be found in the [here](https://www.keycloak.org/docs/latest/server_installation/index.html#_operator).
+The Vibrent documentation can be found [here](https://agile.vignetcorp.com:8086/confluence/display/AC/Keycloak+Operator).
```

Comment:
Perhaps add a comment here about what/why we have a custom version of the keycloak operator even if it is explained in more depth on Confluence.
If/when we push it back upstream, we would have to potentially document whatever the feature changes is that we did.